### DISCUSSION TITLE
Load Balancing with Axum

### DISCUSSION THREAD

**Discussion by chiefofsinners** [tokio-rs/axum#2807] [Q&A] [answered]
### Summary

I'm just wondering whether I'm limited to running Axum on one server? Is it possible to spin up multiple containers that talk to a shared database?

### axum version

0.9.3

**Comment by mladedav** [comment]
You can replicate your app in any way you're used to.

**Comment by dayvejones** [comment, marked as answer]
You can create any desired amount  apps instances. And for balancing, use, for example, Nginx. https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/
